site/content/contribute/webapp/end-to-end-tests.md

```diff
@@ -25,7 +26,30 @@ Not familiar with Cypress? Here is some documentation that will help you get sta
     - Initiate via node CLI to selectively run specs based on test metadata, e.g. `node run_tests.js --group='@accessibility'` which will run all specs with `@accessibility` metadata.
 3.  Tests are executed according to your selection and will display whether the tests passed or failed.
 
+### Environment Variables
+
+We use several environment variables for Cypress testing in order to:
+- Easily change when running in CI.
+- Cater to different values across developer machines.
+
+Environment variables are [defined in cypress.json](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress.json) under the `env` key. In most cases, you don't need to change the values as it makes use of the default local developer setup. However, when you do the easiest method is to override by exporting `CYPRESS_*`, where `*` is the key of the variable, for example: `CYPRESS_adminUsername`. See [Cypress documentation](https://docs.cypress.io/guides/guides/environment-variables.html#Setting) for details.
```

Comment:
```suggestion
Environment variables are [defined in cypress.json](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress.json) under the `env` key. In most cases you don't need to change the values, because it makes use of the default local developer setup. If you do need to make changes, the easiest method is to override by exporting `CYPRESS_*`, where `*` is the key of the variable, for example: `CYPRESS_adminUsername`. See [Cypress documentation](https://docs.cypress.io/guides/guides/environment-variables.html#Setting) for details.
```

Reply:
Done.